src/components/home/styledHome.js

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+
+export const HomeSection = styled.section`
+    display: flex;
+    flex-direction: column;
+    width: 94%;
+    padding: 74px 0 40px 0;
+    margin: auto;
+`
+
+export const Title = styled.h1`
+    width: 60%;
```

Comment:
The width in on the design file is 859px but the width is currently 812px. One solution is to hard code the width and use the br element to make the text look like the design.